.github/workflows/start-release.yml

```diff
@@ -174,10 +170,21 @@ jobs:
             git commit --message "Changelog for ${{inputs.version}}"
             echo "\`CHANGELOG.md\` has been updated on the release branch." >> "$GITHUB_STEP_SUMMARY"
           fi
+      - name: Tag master for next release
+        if: steps.check-existing.outputs.branch-exists == 'false'
```

Comment:
[no-op] I wouldn't say that the condition is the perfect guard here. If making the workflow idempotent, I'd suggest to move the check inside the script and check for presence of a `${{needs.variables.outputs.next-release}}` tag. Similarly, other steps (e.g. `Create and annotate branch ${{needs.variables.outputs.branch}}`) can check for presence of a thing (`x.y.0-rc.0` tag) before adding it.

However, the current step is consistent with other steps so I don't insist on changing that.